### ISSUE TITLE
New /DO switch conflicts with ISPP's /d<name>[=value] option when name begins with an "O"

### ISSUE THREAD

**Issue by ogiesen** [jrsoftware/issrc#161]
As of Inno 5.5.5, the following sample commandline which was formerly a valid way to set an ISPP define will no longer produce any output:

`iscc.exe /dOptionalStuff MyProject.iss`

Should be obvious why. I would suggest that Inno should ignore the switch when there are any other characters after the "O"

**Comment by DeeHants** [comment]
Duplicate of #152 .
This has [already been fixed](https://github.com/jrsoftware/issrc/commit/997f738253ca130e4b61698b20293e3e2bd7b135) and is [waiting on the next release](http://jrsoftware.github.io/issrc/whatsnew.htm).
